build_tools/benchmarks/run_benchmarks.sh

```diff
@@ -59,7 +59,7 @@ elif [[ "${TARGET_DEVICE_NAME}" == "c2-standard-16" ]]; then
         --device_model=GCP-c2-standard-16 \
         --cpu_uarch=CascadeLake \
         --verbose
-elif [[ "${TARGET_DEVICE_NAME}" == "pixel-4" ]]; then
+elif [[ "${TARGET_DEVICE_NAME}" =~ ^(pixel-4|pixel-6-pro|moto-edge-x30)$ ]]; then
```

Comment:
moto edge is still waiting for a runner: https://github.com/openxla/iree/actions/runs/4927200698/jobs/8804442564?pr=13442

Reply:
Fixed the runner configuration. The run is now completed.